src/main/java/uk/gov/ons/census/fwmt/jobservice/service/impl/JobServiceImpl.java

```diff
@@ -46,4 +47,13 @@ public void convertAndSendCreate(CreateFieldWorkerJobRequest jobRequest) throws
     cometRestClient.sendCreateJobRequest(caseRequest, String.valueOf(jobRequest.getCaseId()));
     gatewayEventManager.triggerEvent(String.valueOf(jobRequest.getCaseId()), COMET_CREATE_ACK, LocalTime.now());
   }
+
+  public void convertAndSendCreateCancel(CancelFieldWorkerJobRequest cancelJobRequest) throws GatewayException {
+    final CometConverter cometConverter = cometConverters.get(cancelJobRequest.getCaseId());
```

Comment:
Do not think we need this. It's not using the named component feature of the comet converter where it grabs the survey type and will use the converter of the named survey type. take a look at the create job for more clarification.

Reply:
Updated as requested